src/OrbitGl/App.cpp

```diff
@@ -1279,6 +1283,9 @@ void OrbitApp::LoadSymbols(const std::filesystem::path& symbols_path, ModuleData
        frame_track_function_hashes = std::move(frame_track_function_hashes), module_data,
        symbols_path, this](const ErrorMessageOr<void>& result) {
         if (result.has_error()) {
+          if (crash_handler_ != nullptr) {
+            crash_handler_->DumpWithoutCrash();
+          }
```

Comment:
personal preference: I would put this between `ERROR` and `SendErrorToUi`.

Reply:
Moved DumpWithoutCrash() call there